00-open-a-shapefile.Rmd

```diff
@@ -221,11 +219,11 @@ extent(aoiBoundary)
 ```
 
 #Import a line and point shapefile
-=======
+
 Bonus: For more on parameter options in base r plot() check out these resources:
 
 * <a href="http://www.statmethods.net/advgraphs/parameters.html" target="_blank">Parameter methods in R.</a>
-* <a href="https://codeyarns.files.wordpress.com/2011/07/20110729-vim-named-colors.png?w=700">Color names in R</a>
+* <a href="https://codeyarns.files.wordpress.com/2011/07/20110729-vim-named-colors.png?w=700"target="_blank">Color names in R</a>
```

Comment:
is there a space between 700" and target="_blank" ? i can't tell. there needs to be.